docs/visualization/.pages

```diff
@@ -1,8 +1,20 @@
 nav:
     - Visualization: index.md
-    - ギャラリー: gallery.md
-    - LTTngEventFilterの使用方法: lttng_event_filter.md
-    - treat_drop_as_delayフラグの説明: about_treat_drop_as_delay.md
-    - トラブルシューティング: trouble_shooting.md
-    - Path Verification: path_verify.md
-    - Detailed Behavior Investigation Methods: investigate_behavior.md
+    - Basic APIs Concept: visualization/concept/basic_api_concept
+    - Visualization APIs:
+        - List:
+            - Message flow: visualization/visualization_api/message_flow
+            - Callback Scheduling Visualization: visualization/visualization_api/callback_scheduling_visualization
+            - Chain latency: visualization/visualization_api/chain_latency
+            - Latency time series: visualization/visualization_api/latency_time_series
+            - Latency histogram: visualization/visualization_api/latency_histogram
+            - Callback Information: visualization/visualization_api/callback_information
+            - Publish/Subscription Information: visualization/visualization_api/pub_sub_information
+            - Communication Information: visualization/visualization_api/communication_information
+            - Response Time: visualization/visualization_api/response_time
```

Comment:
@keita1523  
Page structure is not kind for beginner of CARET. For example, `message_flow` is not common word for ROS 2 users.

I suppose that the following page order is appropriate for ROS 2 users who are beginners of CARET.

- Callback
  - Freq/Period/Latency (Execution time)
  - Scheduling
- Communication
  - Freq/Period/Latency (Execution time)
- Path
  - Message flow
  - Response time
  - etc.

Reply:
OK.
I modified the `.page` file and `index.md` like upper order.
I added a Node item.